### ISSUE TITLE
Broken with zio 2.0.0-RC4

### ISSUE THREAD

**Issue by csar** [zio/interop-reactive-streams#313]
```
pub <- stream.toPublisher.debug("publisher")
ev <- pub.toStream().debug("stream").runLast.debug("last")
```

creates the Publisher, but toStream() fails

**Comment by runtologist** [comment]
I assume you are referring to interop-reactive-streams v2.0.0-RC4, which has been compiled against ZIO 2.0.0-RC3. ZIO 2.0.0-RC4 is not binary compatible with ZIO 2.0.0-RC3. 

I have just published interop-reactive-streams v2.0.0-RC5, which is compatible with ZIO 2.0.0-RC4.

**Comment by csar** [comment]
> 

Yes, I saw it last night and it works fine now, you can close this issue

Thx for the speedy resolution!
